indexing-solr/src/main/java/edu/unc/lib/boxc/indexing/solr/filter/SetDescriptiveMetadataFilter.java

```diff
@@ -441,30 +456,36 @@ private String formatName(Element nameEl) {
             if (nameParts.size() == 1) {
                 nameValue = nameParts.get(0).getValue();
             } else if (nameParts.size() > 1) {
-                Element genericPart = JDOMQueryUtil.getElementByAttribute(nameParts, "type", null);
-                if (genericPart != null) {
-                    nameValue = genericPart.getValue();
-                } else {
-                    // If there were multiple non-generic name parts, then try to piece them together
-                    Element givenPart = JDOMQueryUtil.getElementByAttribute(nameParts, "type", "given");
-                    Element familyPart = JDOMQueryUtil.getElementByAttribute(nameParts, "type", "family");
-                    StringBuilder nameBuilder = new StringBuilder();
-                    if (familyPart != null) {
-                        nameBuilder.append(familyPart.getValue());
-                        if (givenPart != null) {
-                            nameBuilder.append(',').append(' ');
-                        }
-                    }
+                Element givenPart = JDOMQueryUtil.getElementByAttribute(nameParts, "type", null);
+                if (givenPart == null) {
```

Comment:
I'd suggest changing these checks to using `StringUtils.isBlank` as well, I was able to get facet entries like ", Given" by doing the following:
```
  <mods:name>
    <mods:namePart type="given">Given</mods:namePart>
    <mods:role>
      <mods:roleTerm>Creator</mods:roleTerm>
    </mods:role>
    <mods:namePart type="family" />
  </mods:name>
```